src/javascript/components/select2-multi-suggest.js

```diff
@@ -93,7 +93,7 @@ export default class Select2MultiSuggest extends React.Component {
    */
   s2TagTemplate(opt) {
     let data = opt.element ? $(opt.element).data() : opt;
-    return `<span title="${data.name}">${data.id}</span>`;
+    return html`<span title="${data.name}">${data.id}</span>`;
```

Comment:
Hmmm, wouldn't it be easier to escape the passed data rather than the whole thing?

Reply:
Yeah, but more verbose. The `html` tag allows us to automatically escape everything that's being substituted, but leave the template parts of the template string unchanged.